DDDEastAnglia/Helpers/Security/IOAuthClientInfo.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using DotNetOpenAuth.AspNet;
+using Microsoft.Web.WebPages.OAuth;
+
+namespace DDDEastAnglia.Helpers.Security
+{
+    public interface IOAuthClientInfo
```

Comment:
I just extracted the interface (I think) to help me with testing. The OAuth code is very messy, and could do with looking at separately.